app/views/component-listing.html

```diff
@@ -38,6 +38,9 @@
 				<tr><th></th><th></th><th>Updated</th></tr>
 			</thead>
 			<tbody>
+			<tr class="component-group empty-search">
```

Comment:
This should be hidden to start with. You could use `aria-hidden="true"` on the `<tr>` which should already have styles set to `display: none`